recipes/numpy/numpy-1.16.4.patch

```diff
@@ -12,15 +13,72 @@ diff -Naur numpy/core/setup.py numpy-1.16.4/numpy/core/setup.py
-
---- numpy-1.16.4.orig/numpy/linalg/setup.py	2019-02-21 17:33:42.000000000 +0100
-+++ numpy-1.16.4/numpy/linalg/setup.py	2019-09-28 17:34:15.000000000 +0200
-@@ -38,9 +38,12 @@
+@@ -822,7 +824,10 @@ def configuration(parent_package='',top_path=None):
+             join('include', 'numpy', 'npy_1_7_deprecated_api.h'),
+             # add library sources as distuils does not consider libraries
+             # dependencies
+-            ] + npysort_sources + npymath_sources
++
++            # XXX This breaks for iOS, it results on duplicate symbols
++            # ] + npysort_sources + npymath_sources
++            ] #+ npysort_sources + npymath_sources
+ 
+     multiarray_src = [
+             join('src', 'multiarray', 'alloc.c'),
+@@ -921,7 +926,7 @@ def configuration(parent_package='',top_path=None):
+ 
+     config.add_extension('_multiarray_umath',
+                          sources=multiarray_src + umath_src +
+-                                 npymath_sources + common_src +
++                                 common_src +
+                                  [generate_config_h,
+                                   generate_numpyconfig_h,
+                                   generate_numpy_api,
+diff --git a/numpy/linalg/lapack_lite/python_xerbla.c b/numpy/linalg/lapack_lite/python_xerbla.c
+index dfc195556..7110d1fc3 100644
+--- a/numpy/linalg/lapack_lite/python_xerbla.c
++++ b/numpy/linalg/lapack_lite/python_xerbla.c
+@@ -20,7 +20,7 @@
+   info: Number of the invalid parameter.
+ */
+ 
+-int xerbla_(char *srname, integer *info)
++int custom_xerbla_(char *srname, integer *info)
+ {
+         static const char format[] = "On entry to %.*s" \
+                 " parameter number %d had an illegal value";
+diff --git a/numpy/linalg/lapack_litemodule.c b/numpy/linalg/lapack_litemodule.c
+index 696a6d874..d187ad2d1 100644
+--- a/numpy/linalg/lapack_litemodule.c
++++ b/numpy/linalg/lapack_litemodule.c
+@@ -45,7 +45,7 @@ extern int FNAME(zungqr)(int *m, int *n, int *k, f2c_doublecomplex a[],
+                           int *lda, f2c_doublecomplex tau[],
+                           f2c_doublecomplex work[], int *lwork, int *info);
+ 
+-extern int FNAME(xerbla)(char *srname, int *info);
++extern int FNAME(custom_xerbla)(char *srname, int *info);
+ 
+ static PyObject *LapackError;
+ 
+@@ -291,7 +291,7 @@ lapack_lite_xerbla(PyObject *NPY_UNUSED(self), PyObject *args)
+ 
+     NPY_BEGIN_THREADS_DEF;
+     NPY_BEGIN_THREADS;
+-    FNAME(xerbla)("test", &info);
++    FNAME(custom_xerbla)("test", &info);
+     NPY_END_THREADS;
+ 
+     if (PyErr_Occurred()) {
+diff --git a/numpy/linalg/setup.py b/numpy/linalg/setup.py
+index 66c07c9e1..817b45501 100644
+--- a/numpy/linalg/setup.py
++++ b/numpy/linalg/setup.py
+@@ -38,9 +38,12 @@ def configuration(parent_package='', top_path=None):
                  return []
              return [all_sources[0]]
  
 +    def get_lapack_lite_sources_ios(ext, build_dir):
-+        return []
++        return [all_sources[0]]
```

Comment:
isn't this ending up behaving the same way as originally before being patched at all?
https://github.com/numpy/numpy/blob/v1.16.4/numpy/linalg/setup.py#L39

Reply:
Correct, fixed